_posts/2019-06-25-dynamic_linear_models_tfprobability/dynamic_linear_models_tfprobability.Rmd

```diff
@@ -0,0 +1,805 @@
+---
+title: "Dynamic linear models with tfprobability"
+description: > 
+   Previous posts featuring tfprobability - the R interface to TensorFlow Probability - have focused on enhancements to deep neural networks (e.g., introducing Bayesian uncertainty estimates) and fitting hierarchical models with Hamiltonian Monte Carlo. This time, we show how to fit time series using dynamic linear models (DLMs), yielding posterior predictive forecasts as well as the smoothed and filtered estimates from the Kálmán filter.
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydana2019dlmtfp
+bibliography: bibliography.bib
+date: 06-24-2019
+categories:
+  - Probability and statistics 
+  - Time Series
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/thumb.png
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+Welcome to the world of _state space models_. ^[For an excellent yet concise introduction to state space models, see chapter 18 of Murphy (2012) [@Murphy-2012].] There is a _latent process_, hidden from our eyes; and there are _observations_ we make about the things it produces. The process evolves due to some hidden logic (_transition model_); and the way it produces the observations follows some hidden logic (_observation model_). There is noise in process evolution, and there is noise in observation. If the transition and observation models both are linear, and process as well as observation noise are Gaussian, we have a _linear-Gaussian state space model_ (SSM). The task is to infer the latent state from the observations. The most famous technique is the _Kálmán filter_.
+
+In practical applications, two characteristics of SSMs are especially attractive. 
```

Comment:
Does SSM always refer to _linear-Gaussian state space model_? If not, I'd introduce the abbreviation with your first mention of SSM.

Reply:
right, I should add "linear-Gaussian" here